test/extensions/tracers/zipkin/config_test.cc

```diff
@@ -20,7 +20,8 @@ TEST(ZipkinTracerConfigTest, ZipkinHttpTracer) {
   std::string valid_config = R"EOF(
   {
     "collector_cluster": "fake_cluster",
-    "collector_endpoint": "/api/v1/spans"
+    "collector_endpoint": "/api/v1/spans",
+    "trace_id_128bit": true
```

Comment:
revert, and make a new v2 style YAML test to exercise the new feature.

Reply:
Have reverted the change. May need discussion on the type of v2 test, based on the outcome of the v1/v2 config discussion above.